lib/wibox/layout/ratio.lua

```diff
@@ -293,14 +293,14 @@ end
 --- Update all widgets to match a set of a ratio.
 -- The sum of before, itself and after must be 1 or nothing will be done.
 --
---@DOC_wibox_layout_ratio_ajust_ratio_EXAMPLE@
+--@DOC_wibox_layout_ratio_adjust_ratio_EXAMPLE@
```

Comment:
If you change the "path" here, you also need to rename `tests/examples/wibox/layout/ratio/ajust_ratio.lua`.